service/task.go

```diff
@@ -840,67 +791,238 @@ func (uis *UIServer) taskModify(w http.ResponseWriter, r *http.Request) {
 }
 
 func (uis *UIServer) testLog(w http.ResponseWriter, r *http.Request) {
+	usr := gimlet.GetUser(r.Context())
+	data := logData{User: usr}
 	vars := gimlet.GetVars(r)
+	raw := (r.FormValue("text") == "true") || (r.Header.Get("Content-Type") == "text/plain")
+
 	logId := vars["log_id"]
+	taskID := vars["task_id"]
+	testName := vars["test_name"]
+	taskExecutionsAsString := vars["task_execution"]
+	taskExec, err := strconv.Atoi(taskExecutionsAsString)
+	grip.Warning(err)
```

Comment:
Is this really just a warning? If it's not a number, isn't it possible it'll end up returning the wrong test results?

Reply:
It was a warning before.

Reply:
Can we just make it return an error?

Reply:
Still have this question.